.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,34 @@
+name: Error Reference Publisher
+on:
+  push:
+    branches:
+      - 'master'
+    paths-ignore:
+      - 'docs/**'
+      - 'mesheryctl/**'
```

Comment:
We will probably want to run this workflow on change to mesheryctl as well.

Reply:
I think we should do this once mesheryctl gets updated with new error codes :thinking:

Reply:
Ok, that's fine. 

For clarity: the error code utility should have idempotent behavior. What harm does enabling it on mesheryctl now cause?

Reply:
Hmm yes, no harm. I mixed things up a bit. We can enable this :P
The utility will just throw `ERRO[0000] Unable to execute root command (open component_info.json: no such file or directory)` and exit with status 0. So the workflow will still continue.